Lib/test/test_platform.py

```diff
@@ -409,6 +413,55 @@ def test_mac_ver_with_fork(self):
             # parent
             support.wait_process(pid, exitcode=0)
 
+    def test_ios_ver(self):
+        result = platform.ios_ver()
+        if sys.platform == "ios":
+            system, release, model, is_simulator = result
+
+            # Result is a namedtuple
+            self.assertEqual(result.system, system)
+            self.assertEqual(result.release, release)
+            self.assertEqual(result.model, model)
+            self.assertEqual(result.is_simulator, is_simulator)
+
+            # We can't assert specific values without reproducing the logic of
+            # ios_ver(), so we check that the values are broadly what we expect.
+
+            # System is either iOS or iPadOS, depending on the test device
+            self.assertIn(system, {"iOS", "iPadOS"})
+
+            # Release is a major.minor specifier
+            major, minor = release.split(".")
```

Comment:
I'm not set up to test this on a real iOS system but I suspect that the value of `release` could be of the form `17.4.1` in which case the assignment here will fail.

Reply:
It will always be numeric, but yes - it can occasionally be a triple.